### ISSUE TITLE
TERRA-70 ⁃ access_list reosource docs asking for keyspace for importing

### ISSUE THREAD

**Issue by bigjew92** [datastax/terraform-provider-astra#147]
Hi there,

Please provide the following details with your issue report.

### Affected Resource(s)
Please list the resources as a list, for example:

* astra_access_list

### Expected Behavior
I do not believe that access_list needs a keyspace and if it does, I'm not sure which one it would be.

### Actual Behavior
The import syntax might be wrong or is just unclear.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:

```
Import is supported using the following syntax:

# the import id includes the database_id and the keyspace name.
terraform import astra_access_list.example 48bfc13b-c1a5-48db-b70f-b6ef9709872b/keyspace/example
```

https://registry.terraform.io/providers/datastax/astra/latest/docs/resources/access_list#import

┆Issue is synchronized with this [Jira Task](https://datastax-oss.atlassian.net/browse/TERRA-70) by [Unito](https://www.unito.io)
┆friendlyId: TERRA-70
┆priority: Major

**Comment by emerkle826** [comment]
@bigjew92 Thanks for reporting this. As it turns out the docs for `access_list` import are incorrect, and the functionality is broken. The docs should state:
```
Import is supported using the following syntax:

# the import id should be the database_id
terraform import astra_access_list.example 48bfc13b-c1a5-48db-b70f-b6ef9709872b
```

I'll use this ticket to fix the docs and fix the functionality.

**Comment by emerkle826** [comment]
@bigjew92 Also, this was fixed in v2.1.1. Import was broken prior to this version.

**Comment by bigjew92** [comment]
Gotcha. Thank you for the fix. confirmed I was able to import my access_list to the desired database.
